homeassistant/components/lamarzocco/entity.py

```diff
@@ -0,0 +1,91 @@
+"""Base class for the La Marzocco entities."""
+
+import asyncio
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, UPDATE_DELAY
+from .coordinator import LmApiCoordinator
+
+
+@dataclass
+class LaMarzoccoEntityDescriptionMixin:
+    """Mixin for all LM entities."""
+
+    extra_attributes: dict
```

Comment:
This PR is huge and introduces many features, which are currently not used.
To reduce the PR size and we also don't accept unused code, please remove all feature, which are not used in this PR. These features can be added in a later PR, if required

```suggestion
```